### DISCUSSION TITLE
Tallying Algorithm and how proportional consensus is determined?

### DISCUSSION THREAD

**Discussion by themeremirror** [pik-gane/vodle#271] [Q&A] [answered]
Hi Folks, 

My name is Will, I'm a Masters student with a research centre at the University of Wollongong in Australia. I'm working on a report for some researchers who use deliberative democratic methods here. I'm very interested in the work you folks are doing and I'm planning to include Vodle in this report as a potential tool for them and hoped someone might be able to give me a little more context on a few things.

**Question:**

 I was hoping to get a little more context on how Vodle is determining the 'winnning' outcome with the tallying algorithm. I've had a look through the glossary but still have a few questions. Particularly related to this:
 
 _After a poll of type "winner" has [closed](https://github.com/pik-gane/vodle/blob/main/documentation/development/GLOSSARY.md#closing), the final result of the poll is a single option as the ["winner"](https://github.com/pik-gane/vodle/blob/main/documentation/development/GLOSSARY.md#winner) of the poll.
This option is either the option that received a share of 100%, if such an option exists, or it is determined by drawing from the probability distribution defined by the shares. In other words, the probability that option X wins equals X's share. vodle performs this draw only once as soon as the poll is closed. Although there is no central server that does this draw but the draw is performed by each voter's vodle app, we have made sure that all voters see the same winner. This is achieved by using a pseudo-random number generator that is based on the [poll database's](https://github.com/pik-gane/vodle/blob/main/documentation/development/GLOSSARY.md#poll-database) revision number of the poll's ["state" data item](https://github.com/pik-gane/vodle/blob/main/documentation/development/GLOSSARY.md#poll-data-item), which is the same for all voters and cannot be predicted before the poll's state is changed from running to closed._

If we have 3 options. X, Y and Z with 10, 40 and 50% share respectively does this mean that: 

1. Option X will be the 'winning' outcome 10% of the time
2. Option Y will be the 'winning' outcome 40% of the time
3. Option Z will be the 'winning' outcome 50% of the time

Is it that simple or have I interpreted this incorrectly?

Thanks in advance

**Comment by mensch72** [comment, marked as answer]
Yes, this is correct.
